qrenderdoc/Windows/PipelineState/VulkanPipelineStateViewer.cpp

```diff
@@ -2614,12 +2614,32 @@ void VulkanPipelineStateViewer::setState()
         {
           slotname = QFormatStr("Color %1").arg(a.localIdx);
 
+          // With dynamic rendering, the API references the framebuffer index everywhere, for
+          // example when specifying blend state for attachments or with vkCmdClearAttachments. As
+          // such, RenderDoc shows the same index in Color attachments (i.e. fbIdx == localIdx) to
+          // avoid confusion, even when VK_KHR_dynamic_rendering_local_read maps these attachments
+          // to different "locations" used by the shader.  In that case, the mapped location is
+          // shown besides the attachment index.
+          uint32_t location = a.localIdx;
+          if(a.fbIdx < rp.colorAttachmentLocations.count())
+          {
+            location = rp.colorAttachmentLocations[a.fbIdx];
+            if(location == rp.AttachmentUnused)
```

Comment:
One more minor nit, accessing a constant like this makes it read like it's a real member which could be confusing. I'd rather you do explicitly `VKPipe::RenderPass::AttachmentUnused` to be very clear that it's a constant.

Reply:
Done